tests/suites/cloud/tests/preload/index.js

```diff
@@ -2,7 +2,25 @@
 module.exports = {
   title: "Image preload test",
   run: async function (test) {
+
+    // if test is being done on physical DUT via the testbot, check that the preloaded application is working
+    if(this.workerContract.workerType !== `qemu`){
+      // we should be able to see the app starting. ( TODO: VERIFY THE WEB PAGE IS THE RIGHT ONE)
```

Comment:
Couldn't this be done with a curl and a grep?

Reply:
yeah you're right -- will make the change